firefox-ios/Client/Coordinators/TabTray/TabTrayCoordinator.swift

```diff
@@ -33,6 +33,11 @@ class TabTrayCoordinator: BaseCoordinator,
         initializeTabTrayViewController(selectedTab: tabTraySection)
     }
 
+    func dismissChildTabTrayPanels() {
+        let childVCs: [UIViewController] =  tabTrayViewController.childPanelControllers.flatMap { $0.viewControllers }
```

Comment:
little nit, looks like there's an extra space on the rhs of the equals here

Reply:
Thank you, fixed